apps/betterangels-backend/common/graphql/types.py

```diff
@@ -1,20 +1,35 @@
 from datetime import datetime
-from typing import Optional, Union
+from typing import Any, Optional, Union
```

Comment:
Hi @davelindqvist 

I confirmed the following:
- I confirmed that the "Enter a 10-digit phone number without space or special characters" when the number was 9 digits, or it had a special character.
- I confirmed that it doesn't let me add more than 10 digits.

One thing I noticed
- I didn't see the message "The phone number is not a valid US number." when I entered 10 digits but gibberish number such as 1234567890 and tapped on Update button.

Could you check on this one? Thanks!